model/context.go

```diff
@@ -203,6 +230,10 @@ func (page *Page) Fields() common.MapStr {
 		utility.Set(fields, "url", page.URL.Original)
 	}
 	utility.Set(fields, "referer", page.Referer)
+	utility.Set(fields, "system.cpu.cores", page.Cores)
+	utility.Set(fields, "system.memory.total", page.Memory)
+	utility.Set(fields, "servedViaServiceWorker", page.ServedViaServiceWorker)
+	utility.Set(fields, "networkInfo", page.NetworkInfo.fields())
```

Comment:
As mentioned on https://github.com/elastic/apm-server/issues/3657#issuecomment-615058295, networkInfo feels to me like extensions of the "network" ECS fieldset. Do you see any problems with that?

e.g. networkInfo.type -> network.physical (or something like that; there's precedent in "network.transport")

Reply:
no, no problem. but i understand right `network` should still be under `transaction.device`, and not as top level field?

Reply:
No, where I talked about "device" I was just talking concepts. The recorded fields should be top-level.